src/routeTree.gen.ts

```diff
@@ -13,45 +13,39 @@ import { createFileRoute } from '@tanstack/react-router'
 // Import Routes
 
 import { Route as rootRoute } from './routes/__root'
+import { Route as LoginImport } from './routes/login'
+import { Route as AdminXpubImport } from './routes/_admin.xpub'
 
 // Create Virtual Routes
 
-const XpubsLazyImport = createFileRoute('/xpubs')()
-const LoginLazyImport = createFileRoute('/login')()
 const AboutLazyImport = createFileRoute('/about')()
-const IndexLazyImport = createFileRoute('/')()
 
 // Create/Update Routes
 
-const XpubsLazyRoute = XpubsLazyImport.update({
-  path: '/xpubs',
-  getParentRoute: () => rootRoute,
-} as any).lazy(() => import('./routes/xpubs.lazy').then((d) => d.Route))
-
-const LoginLazyRoute = LoginLazyImport.update({
-  path: '/login',
-  getParentRoute: () => rootRoute,
-} as any).lazy(() => import('./routes/login.lazy').then((d) => d.Route))
-
 const AboutLazyRoute = AboutLazyImport.update({
   path: '/about',
   getParentRoute: () => rootRoute,
 } as any).lazy(() => import('./routes/about.lazy').then((d) => d.Route))
 
-const IndexLazyRoute = IndexLazyImport.update({
-  path: '/',
+const LoginRoute = LoginImport.update({
+  path: '/login',
+  getParentRoute: () => rootRoute,
+} as any)
```

Comment:
Can we somehow get rid of this `as any` part?

Reply:
No. It's auto-generated code